docs/concepts.md

```diff
@@ -26,10 +26,16 @@ on the standard Config class if nothing is found in the database.
 
 You can use your own models to handle user persistence. Shield calls this the "User Provider" class. A default model
 is provided for you at `CodeIgniter\Shield\Models\UserModel`. You can change this in the `Config\Auth->userProvider` setting.
-The only requirement is that your new class MUST extend the provided `UserModel`.
+The only requirement is that your new class MUST extend the provided `UserModel`. Shield has a CLI command to quickly create a custom `MyUserModel` by running the following command in terminal:
+
+```console
+php spark shield:model MyUserModel
+```
+
+You should set `Config\Auth::$userProvider` as follows:
 
 ```php
-public $userProvider = 'CodeIgniter\Shield\Models\UserModel';
+public $userProvider = 'App\Models\MyUserModel';
```

Comment:
Use `::class` for better IDE and SA support.
```suggestion
public $userProvider = \App\Models\MyUserModel::class;
```

Reply:
Therefore, it should be changed in the following cases.

https://github.com/codeigniter4/shield/blob/96f9987d602138c5437f5d9f3d94503ef190eec1/src/Config/Auth.php#L60-L61

https://github.com/codeigniter4/shield/blob/96f9987d602138c5437f5d9f3d94503ef190eec1/src/Config/Auth.php#L343

Reply:
Yes, that would be better.